### PR TITLE
Updated strings.xml for minor copy changes

### PR DESCRIPTION
**Pull request by riyapenn-signal**
### First time contributor checklist

- [X] I have read [how to contribute](https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [X] I have signed the [Contributor License Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [X] I am following the [Code Style Guidelines](https://github.com/signalapp/Signal-Android/wiki/Code-Style-Guidelines)
- [ ] I have tested my contribution on these devices:
 * Device A, Android X.Y.Z
 * Device B, Android Z.Y
 * Virtual device W, Android Y.Y.Z
- [X] My contribution is fully baked and ready to be merged as is
- [ ] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)

----------

### Description

- Permission prompt to include text about backups
- Updated support email domain
- Updated domain to signal.org in a link
- Editted troubleshooting typo

### CODE REVIEW

**Comment by haffenloher** [comment]
I may be wrong about this, but I think the names of the strings need to be changed as well or the translators won't see that they need to re-translate them.

**Comment by greyson-signal** [comment]
@haffenloher Hard to find specific documentation on this, but [this section](https://docs.transifex.com/client/push/#how-source-string-updates-are-handled) implies that modified strings are considered different. Let me know if you find anything that says otherwise :)

**Comment by greyson-signal** [comment]
Merged into 4.22

File: res/values/strings.xml [hunk 1]
```diff
@@ -555,13 +555,13 @@
...
     <string name="RegistrationActivity_i_understand">I understand</string>
     <string name="RegistrationActivity_play_services_error">Play Services Error</string>
     <string name="RegistrationActivity_google_play_services_is_updating_or_unavailable">Google Play Services is updating or temporarily unavailable. Please try again.</string>
     <string name="RegistrationActivity_more_information">More information</string>
     <string name="RegistrationActivity_less_information">Less information</string>
-    <string name="RegistrationActivity_signal_needs_access_to_your_contacts_and_media_in_order_to_connect_with_friends">Signal needs access to your contacts and media in order to connect with friends, exchange messages, and make secure calls</string>
+    <string name="RegistrationActivity_signal_needs_access_to_your_contacts_and_media_in_order_to_connect_with_friends">Signal needs access to your contacts and media in order to connect with friends or restore from a backup.</string>
```
[the remark this answers is not in the document]
Comment (greyson-signal) [reply res/values/strings.xml]: Done.

**Comment by RiseT** [comment]
@haffenloher @greyson-signal No, string names don't have to change in order to get noticed by Transifex. The new strings are available on Transifex now.

**Pull request closed without merging**